modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/22_react-accessing-global-state-inside-of-async-thunks-with-typescript.md

```diff
@@ -21,7 +21,7 @@ export const checkoutCart = createAsyncThunk("cart/checkout", async (_, thunkAPI
 
 </TimeStamp>
 
-<TimeStamp start="0:50" end="0:00">
+<TimeStamp start="0:50" end="1:05">
```

Comment:
There were a couple of these. Went through all of the notes and got these fixed.